fw/face/ndnlp-link-service.go

```diff
@@ -442,3 +442,10 @@ func (op *NDNLPLinkServiceOptions) Flags() (ret uint64) {
 	}
 	return
 }
+
+// Reads a packet without validating the internal fields
+func ReadPacketFast(reader enc.ParseReader) (*spec.Packet, error) {
```

Comment:
I think it should be better named `ReadPacketUnverified` or `ReadPacketRaw`.
The word `Fast` make people think it is a faster alternative with no cost.